eosio.system/src/producer_pay.cpp

```diff
@@ -95,10 +95,10 @@ namespace eosiosystem {
          auto to_per_vote_pay    = to_producers - to_per_block_pay;
 
          INLINE_ACTION_SENDER(eosio::token, issue)( N(eosio.token), {{N(eosio),N(active)}},
-                                                    {N(eosio), asset(new_tokens), std::string("issue tokens for producer pay and savings")} );
+                                                    {N(eosio), asset(to_producers), std::string("issue tokens for producer pay and savings")} );
```

Comment:
There is no token for savings:
"issue tokens for producer pay and savings" --> "issue tokens for producer pay"